src/app/common/stores/applications/Distribution/Wallet.store.ts

```diff
@@ -77,8 +80,10 @@ class WalletStore extends BaseStore implements Wallet {
     if (this.onboardSdk) return;
     this.onboardSdk = initOnboard();
 
-    const wallets = this.onboardSdk.state.select('wallets');
+    const wallets = this.onboardSdk.state.select();
     wallets.subscribe(async (update: any) => {
+      console.warn('Wallet subscription data:', update);
```

Comment:
remove console

Reply:
I added it by purpose, I need to see what is returned on stage and prod when and if it won't work.